redis/commands.cc

```diff
@@ -53,6 +53,22 @@ future<redis_message> get::execute(service::storage_proxy& proxy, redis::redis_o
     });
 }
 
+shared_ptr<abstract_command> exists::prepare(service::storage_proxy& proxy, request&& req) {
+    if (req.arguments_size() != 1) {
+        throw wrong_arguments_exception(1, req.arguments_size(), req._command);
+    }
+    return seastar::make_shared<exists> (std::move(req._command), std::move(req._args[0]));
+}
+
+future<redis_message> exists::execute(service::storage_proxy& proxy, redis::redis_options& options, service_permit permit) {
+    return redis::read_strings(proxy, options, _key, permit).then([] (auto result) {
+        if (result->has_result()) {
+            return redis_message::one();
```

Comment:
According to https://redis.io/commands/exists you should return the number of keys that exist, not 0/1.